web_widget_table/static/js/web_widget_table.js

```diff
@@ -0,0 +1,56 @@
+odoo.define('web_widget_table', function(require)
+{
+    var registry = require('web.field_registry'),
+        AbstractField = require('web.AbstractField'),
+        FieldDataTable = AbstractField.extend({
+        className: 'oe_form_field_table',
+        supportedFieldTypes: ['char'],
+        init: function()
+        {
+            this._super.apply(this, arguments);
+        },
+        _render: function()
+        {
+            this.$el.empty(); 
+            var field = this.nodeOptions['field_name'],
+                fieldModel = this.record.data[field].model,
+                fieldRecordIds = this.record.data[field].res_ids,
+                table = `<table><thead><tr>`,
+                table_fields = this.nodeOptions['fields'],
+                headers = this.nodeOptions['headers'];
+                datatable_params = this.nodeOptions['datatable_params'];
+
+            headers.forEach(element => {
```

Comment:
Odoo 11.0 doesn't support ES6, more info: https://github.com/OCA/odoo-community.org/issues/37

Reply:
I did the changes, thank you